services/libs/tinybird/pipes/search_volume.pipe

```diff
@@ -0,0 +1,32 @@
+DESCRIPTION >
+    Serves the project search volume data for the `Search queries` widget
+
+NODE searchVolume_pipe
+SQL >
+    %
+    SELECT projectId, keyword, dataTimestamp, volume, updatedAt
+    FROM searchVolume
+    WHERE
+        1 = 1
+        {% if defined(startDate) %}
+            AND searchVolume.dataTimestamp
+            >= {{
+                DateTime(
+                    startDate,
+                    description="Select records with dataTimestamp only after the given date",
+                    required=False,
+                )
+            }}
+        {% end %}
+        {% if defined(endDate) %}
+            AND searchVolume.dataTimestamp
+            <= {{
+                DateTime(
+                    endDate,
+                    description="Select records with dataTimestamp only before the given date",
+                    required=False,
+                )
+            }}
+        {% end %}
+        {% if defined(projectId) %} AND searchVolume.projectId = {{ projectId }} {% end %}
```

Comment:
NIT: It's gonna help with consistency and usability of the pipe if we provide a project slug filter instead of an opaque id filter here. (since everywhere else FE expects this filtering to be done by slug) 

There's a reusable example [here](https://github.com/CrowdDotDev/crowd.dev/blob/e8dc098313ccef3e6a0d6f71fe0b84345e640bd4/services/libs/tinybird/pipes/package_downloads.pipe)
Another benefit of using `segments_filtered` pipe for this is that it provides `insightsProjects.enabled` check as well